scheds/rust/scx_lavd/src/bpf/intf.h

```diff
@@ -118,7 +118,7 @@ struct task_ctx {
 	u64	wait_freq;		/* waiting frequency in a second */
 	u64	wake_freq;		/* waking-up frequency in a second */
 	u64	svc_time;		/* total CPU time consumed for this task scaled by task's weight */
-	u64	dsq_id;			/* DSQ id where a task run for statistics */
+	u32	prev_cpu_id;		/* where a task ran last time */
```

Comment:
it might make sense to document the difference between `prev_cpu_id` and `cpu_id`.